test/e2e/e2e_test.go

```diff
@@ -33,10 +33,14 @@ import (
 	// "github.com/onsi/ginkgo"
 
 	"k8s.io/component-base/version"
+	conformancetestdata "k8s.io/kubernetes/test/conformance/testdata"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/config"
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
-	"k8s.io/kubernetes/test/e2e/generated"
+	e2etestingmanifests "k8s.io/kubernetes/test/e2e/testing-manifests"
+	e2enodetestingmanifests "k8s.io/kubernetes/test/e2e_node/testing-manifests"
```

Comment:
does `test/e2e` depend on `test/e2e_node` today? should it?

Reply:
`test/e2e` shouldn't depend on `test/e2e_node`. I am dropping that import.

Reply:
I rechecked the file references in `test/e2e` and couldn't find any references to the manifests in `test/e2e_node/testing-manifests`.